README.rst

```diff
@@ -95,38 +93,33 @@ directly, but instead be manually inspected. E.g.:
        fiel->feel, field, file, phial,
 
    Note the last comma! You need to use it, otherwise the last suggestion
-   will be discarded (see below for why). When there is more than one
+   will be discarded (for historic reasons). When there is more than one
```

Comment:
Is now the time to remove the restriction or re-implement this feature (or are we worried about people running new dictionaries against old code? I guess it's not too arduous a requirement to keep for future re-implementation.

Reply:
It's possible that people have their own custom dictionaries and it would be good not to break them or force them to update them

Reply:
Good point. We could add it back, although I guess if you're using e.g. a code word, you hopefully know what it is and why. I guess showing which dictionary it's come from is more initially valuable. This is just spit-balling anyway.